src/popover/popover.spec.ts

```diff
@@ -508,6 +509,22 @@ describe('ngb-popover', () => {
          expect(windowEl.textContent.trim()).toBe('Great tip!');
        }));
 
+    it('should modify the popper options', (done) => {
+      const fixture = createTestComponent(`<div ngbPopover="Great tip!" placement="right">Trigger</div>`);
+      const popover = fixture.debugElement.query(By.directive(NgbPopover)).injector.get(NgbPopover);
+
+      const spy = createSpy();
+      popover.popperOptions = (options: Partial<Options>) => {
```

Comment:
I wonder if we should explain how to use if more clearly in docs:
- is the initial config readonly or can be modified?
- should the user clone it?
- will initial config be discarded and ignored?